diff --git a/apple/scripts/build-xcframework.sh b/apple/scripts/build-xcframework.sh
index 56bcb4ced0c25da2aaa170e3d2657ae2e4798ca7..45e7d2efc56f8944d7bd0f61ba9db33ece09a6f4 100755
--- a/apple/scripts/build-xcframework.sh
+++ b/apple/scripts/build-xcframework.sh
@@ -229,6 +229,8 @@ build_slice() {
     DEBUG_INFORMATION_FORMAT=dwarf-with-dsym \
     COMPILER_INDEX_STORE_ENABLE=NO \
     SWIFT_COMPILATION_MODE=wholemodule \
+    CODE_SIGNING_ALLOWED=NO \
+    CODE_SIGNING_REQUIRED=NO \
   )
 
   local product_path="${BUILD_PRODUCTS_PATH}/${build_dir_name}"
